compute/client_library/ingredients/disks/create_secondary_custom.py

```diff
@@ -0,0 +1,75 @@
+#  Copyright 2024 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# flake8: noqa
+
+from google.cloud import compute_v1
+
+
+# <INGREDIENT create_secondary_custom_disk>
+def create_secondary_custom_disk(
+    primary_disk_name: str,
+    primary_disk_project: str,
+    primary_disk_zone: str,
+    secondary_disk_name: str,
+    secondary_disk_project: str,
+    secondary_disk_zone: str,
+    disk_size_gb: int,
+    disk_type: str = "pd-ssd",
+) -> compute_v1.Disk:
+    """Creates a custom secondary disk whose properties differ from the primary disk.
+    Args:
+        primary_disk_name (str): The name of the primary disk.
+        primary_disk_project (str): The project of the primary disk.
+        primary_disk_zone (str): The location of the primary disk.
+        secondary_disk_name (str): The name of the secondary disk.
+        secondary_disk_project (str): The project of the secondary disk.
+        secondary_disk_zone (str): The location of the secondary disk.
+        disk_size_gb (int): The size of the disk in GB. Should be the same as the primary disk.
+        disk_type (str): The type of the disk. Must be one of pd-ssd or pd-balanced.
+    """
+    disk_client = compute_v1.DisksClient()
+    disk = compute_v1.Disk()
+    disk.name = secondary_disk_name
+    disk.size_gb = disk_size_gb
+    disk.type = f"zones/{primary_disk_zone}/diskTypes/{disk_type}"
+    disk.async_primary_disk = compute_v1.DiskAsyncReplication(
+        disk=f"projects/{primary_disk_project}/zones/{primary_disk_zone}/disks/{primary_disk_name}"
+    )
+
+    # Specify additional guest OS features for the secondary disk
+    # Set to one or more of the following values: - VIRTIO_SCSI_MULTIQUEUE - WINDOWS - MULTI_IP_SUBNET
+    # - UEFI_COMPATIBLE - GVNIC - SEV_CAPABLE - SUSPEND_RESUME_COMPATIBLE
+    # - SEV_LIVE_MIGRATABLE_V2 - SEV_SNP_CAPABLE - TDX_CAPABLE - IDPF
```

Comment:
Is this correct formatting? Seems like the list should be one element per line :)

Reply:
And is there anywhere where one can read more about those features? If so, please leave a link. For example, I have no idea what TDX_CAPABLE means

Reply:
I changed the comment and provide link with detailed description of each feature 🙌 PTAL